docs/graphql-api.md

```diff
@@ -1,11 +1,11 @@
 # 1 Queries
 # 1.1 Basics
-For each type `Entity` which you define in your schema, an `entity` and `entities` field will be generated on the top-level `Query` type.
+For each type `Entity` which you define in your schema, an `entity` and `entities` field will be generated on the top-level `Query` type. Note that `query` does not need to be included at the top of the `graphql` query when using The Graph. 
```

Comment:
`which you define` -> `that you define`

Reply:
Drop the `Now, an `entity` is what you will define in order to detemine how the `data source` will be indexed in a Graph Node.`. That's just how GraphQL works, nothing specific here. There's a typo in the sentence anyway (`detemine`).

Reply:
looks like the 2nd comment got copy pasted from above, that sentance was in a different file. I fixed the 1st one though